tests/test_bulk_rx_basic.py

```diff
@@ -36,8 +46,38 @@ def do_test(arch, clk, phy, data_valid_count, usb_speed, seed):
 
         dataval += pktlength
 
-    do_usb_test(arch, clk, phy, usb_speed, packets, __file__, seed, level='smoke', extra_tasks=[])
+    tester = do_usb_test(arch, clk, phy, usb_speed, packets, __file__, seed, level='smoke', extra_tasks=[])
+    
+    return tester
+
+def run_on(**kwargs):
+
+    for name,value in kwargs.items():
+        arg_value = args.get(name)
+        if arg_value is not None and value != arg_value:
+            return False
+
+    return True
+
+@pytest.fixture
+def runall_rx(capfd):
+    testname,extension = os.path.splitext(os.path.basename(__file__))
```

Comment:
The contents of this function is duplicated also. While I'm not sure you can defined a fixture in another file, you can certainly define a function to return most of this information in one place, and use that in the all the test fixture definitions.